mitiq/lre/tests/test_layerwise_folding.py

```diff
@@ -30,36 +32,41 @@
 test_circuit1_with_measurements.append(ops.measure_each(*qreg1))
 
 
-def test_multivariate_layerwise_scaling():
+@pytest.mark.parametrize("circuit_type", SUPPORTED_PROGRAM_TYPES.keys())
```

Comment:
Related to a comment I left below: better to use `SUPPORTED_PROGRAM_TYPES` as parameter.